sdk/inc/azure/core/az_mqtt5.h

```diff
@@ -255,6 +262,46 @@ typedef struct
   int32_t id;
 } az_mqtt5_suback_data;
 
+/**
+ * @brief MQTT 5 unsubscribe data.
+ *
+ */
+typedef struct
+{
+  /**
+   * @brief The properties of unsubscribe request.
+   *
+   * @details Set to NULL if no properties are required.
+   */
+  az_mqtt5_property_bag* properties;
+  /**
+   * @brief Topic filter to unsubscribe from.
+   */
+  az_span topic_filter;
+
+  /**
+   * @brief Id to correlate the unsubscribe request with the response acknowledgement.
+   */
+  int32_t out_id;
```

Comment:
out?

Reply:
The MQTT stack will set this variable when executing an unsubscribe, the application can then use it to correlate messages.